pkg/genericapiserver/genericapiserver_test.go

```diff
@@ -267,10 +262,11 @@ func (authn *mockAuthenticator) AuthenticateRequest(req *http.Request) (user.Inf
 // TestInstallSwaggerAPI verifies that the swagger api is added
 // at the proper endpoint.
 func TestInstallSwaggerAPI(t *testing.T) {
-	server, etcdserver, _, assert := setUp(t)
+	etcdserver, _, assert := setUp(t)
 	defer etcdserver.Terminate(t)
 
 	mux := http.NewServeMux()
+	server := GenericAPIServer{}
```

Comment:
mostly for equivalence with previous code, and not being sure where it is passed or the side effects of the functions called on it